cephfs/cephfs_test.go

```diff
@@ -15,54 +13,21 @@ import (
 )
 
 var (
-	CephMountDir     = "/tmp/ceph/mds/mnt/"
-	requireCephMount = false
-	testMdsName      = "Z"
+	CephMountDir = "/tmp/ceph/mds/mnt/"
+	testMdsName  = "Z"
 )
 
 func init() {
 	mdir := os.Getenv("GO_CEPH_TEST_MOUNT_DIR")
 	if mdir != "" {
 		CephMountDir = mdir
 	}
-	reqMount := os.Getenv("GO_CEPH_TEST_REQUIRE_MOUNT")
-	if reqMount == "yes" || reqMount == "true" {
-		requireCephMount = true
-	}
 	mdsName := os.Getenv("GO_CEPH_TEST_MDS_NAME")
 	if mdsName != "" {
 		testMdsName = mdsName
 	}
```

Comment:
dito

Reply:
I'm not clear on this but you seem to be suggesting GO_CEPH_TEST_MDS_NAME/testMdsName  can be removed. However, this is used by the mds_command function tests over in command_test.go. If I'm wrong just feel free to ignore me.